bugwarrior/services/gitlab.py

```diff
@@ -433,7 +434,19 @@ def include_todo(todo):
 
     def issues(self):
         tmpl = '{scheme}://{host}/api/v4/projects'
-        all_repos = self._fetch_paged(tmpl)
+
+        all_repos = []
+        if self.include_repos:
+            for repo in self.include_repos:
```

Comment:
You are using `include_repos` here, however, there is also `include_regex`, which you should check on as well.

Reply:
To allow `include_regex` would require fetching all the projects still; is that a correct statement?

Reply:
Possibly. Probably the best solution would be to find a GitLab APIv4 call that returns a list of repos based on a regex, but I don't know if there is such a call. Might be worth suggesting that to the GitLab API devs, this would save projects like bugwarrior using the API a lot of time.

Reply:
I've added code to handle the `include_regex` functionality in 04f6e2d

Reply:
Gitlab API allows for searching projects, but not by regex; I'll investigate opening an issue with their core functionality to see if they'd be willing to provide that.